Tests/Singulink.Globalization.Currency.Tests/SortedMoneySetTests/RemoveAllTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Shouldly;
+
+namespace Singulink.Globalization.Tests.SortedMoneySetTests;
+
+[TestClass]
+public class RemoveAllTests
+{
+    private static readonly Money _usd100 = new(100m, "USD");
+    private static readonly Money _cad50 = new(50m, "CAD");
+    private static readonly Money _eur25 = new(25m, "EUR");
+    private static readonly ImmutableSortedMoneySet _immutableSet = new(_usd100, _cad50, _eur25);
+    private readonly SortedMoneySet _set = _immutableSet.ToSet();
+
+    [TestMethod]
+    public void RemoveAll_AllExistingCurrenciesFromSet_IsSuccessful()

```

Comment:
These don't need the first test name prefix, they are all the same

Reply:
Taken care of.